apps/server/src/api-integration/integration.controller.ts

```diff
@@ -156,35 +150,32 @@ const actionHandlers: Record<string, ActionHandler> = {
   },
   /* Extra timers */
   extratimer: (payload) => {
-    if (payload && typeof payload === 'string') {
-      if (payload === SimplePlayback.Start) {
-        const reply = extraTimerService.start();
-        return { payload: reply };
-      }
-      if (payload === SimplePlayback.Pause) {
-        const reply = extraTimerService.pause();
-        return { payload: reply };
-      }
-      if (payload === SimplePlayback.Stop) {
-        const reply = extraTimerService.stop();
-        return { payload: reply };
-      }
-    }
-
     if (payload && typeof payload === 'object') {
-      if ('settime' in payload) {
-        const time = numberOrError(payload.settime);
-        const reply = extraTimerService.setTime(time);
-        return { payload: reply };
-      }
-      if ('direction' in payload) {
-        if (payload.direction === SimpleDirection.CountUp || payload.direction === SimpleDirection.CountDown) {
-          const reply = extraTimerService.setDirection(payload.direction);
-          return { payload: reply };
-        } else {
-          throw new Error('Invalid direction payload');
+      Object.entries(payload).forEach(([key, value]) => {
+        if ('settime' in value) {
+          const time = numberOrError(value.settime);
+          extraTimerService[key].setTime(time);
         }
-      }
+        if ('direction' in value) {
+          if (value.direction === SimpleDirection.CountUp || value.direction === SimpleDirection.CountDown) {
+            extraTimerService[key].setDirection(value.direction);
+          }
+        }
+        if ('playback' in value) {
+          switch (value.playback) {
+            case SimplePlayback.Start:
+              extraTimerService[key].start();
```

Comment:
I think the service should maintain many timers. And take as an argument which timer (index?) the command is destined to

We could also make a small object that maps the enum keys to actions to help here